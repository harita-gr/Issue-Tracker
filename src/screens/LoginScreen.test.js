// import React from 'react';
// import {mount } from 'enzyme';
// import LoginScreen from './LoginScreen'

// describe('<LoginScreen/>', () => {

//     let wrapper;

//     // beforeEach(() =>{
        
//     //  });

//         // it('render the correct heading name',() => {
//         //     wrapper = mount(<LoginScreen/>); 
//         //     expect(wrapper.find("h3").text()).toContain("WELCOME BACK USER")
//         // })

//         // it('User login',() => {
//         //     wrapper = mount(<LoginScreen/>); 
//         //     expect(wrapper.find("login-btn").simulate('click')
//         // })

// });