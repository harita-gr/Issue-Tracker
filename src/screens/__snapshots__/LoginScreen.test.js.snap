// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginScreen/> render the login form correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <div
          class="row"
        >
          
          <div
            class="col-sm-12"
          >
            <h3
              align="center"
              class="pt-2"
            >
               WELCOME BACK USER
            </h3>
          </div>
          <div
            class="mx-auto col-lg-4 col-md-6 col-sm-12"
          >
            <form
              class=""
            >
              <div
                class="form-group"
              >
                <label
                  class="form-label"
                >
                  Email address
                </label>
                <input
                  class="form-control"
                  id="email"
                  name="email"
                  placeholder="Enter email"
                  type="email"
                  value=""
                />
              </div>
              <div
                class="form-group"
              >
                <label
                  class="form-label"
                >
                  Password
                </label>
                <input
                  class="form-control"
                  id="password"
                  name="password"
                  placeholder="Password"
                  type="password"
                  value=""
                />
              </div>
              <button
                class="login-btn btn btn-primary"
                type="submit"
              >
                Submit
              </button>
            </form>
          </div>
          <div
            class="col-sm-12"
          >
            <h5
              align="center"
              class="pt-2"
            >
              New Customer? 
              <a
                href="/register"
              >
                SIGN UP
              </a>
            </h5>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <div
        class="row"
      >
        
        <div
          class="col-sm-12"
        >
          <h3
            align="center"
            class="pt-2"
          >
             WELCOME BACK USER
          </h3>
        </div>
        <div
          class="mx-auto col-lg-4 col-md-6 col-sm-12"
        >
          <form
            class=""
          >
            <div
              class="form-group"
            >
              <label
                class="form-label"
              >
                Email address
              </label>
              <input
                class="form-control"
                id="email"
                name="email"
                placeholder="Enter email"
                type="email"
                value=""
              />
            </div>
            <div
              class="form-group"
            >
              <label
                class="form-label"
              >
                Password
              </label>
              <input
                class="form-control"
                id="password"
                name="password"
                placeholder="Password"
                type="password"
                value=""
              />
            </div>
            <button
              class="login-btn btn btn-primary"
              type="submit"
            >
              Submit
            </button>
          </form>
        </div>
        <div
          class="col-sm-12"
        >
          <h5
            align="center"
            class="pt-2"
          >
            New Customer? 
            <a
              href="/register"
            >
              SIGN UP
            </a>
          </h5>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
